src/vs/workbench/services/workingCopy/common/workingCopyService.ts

```diff
@@ -138,6 +139,11 @@ export class WorkingCopyService extends Disposable implements IWorkingCopyServic
 	registerWorkingCopy(workingCopy: IWorkingCopy): IDisposable {
 		const disposables = new DisposableStore();
 
+		// {{SQL CARBON EDIT}} @chlafreniere need to block working copies of notebook editors from being tracked
+		if (workingCopy.resource.path.includes('notebook-editor-') && workingCopy.resource.scheme === Schemas.untitled) {
```

Comment:
Why only untitled here?

Reply:
Each notebook cell is an untitled editor: https://github.com/Microsoft/azuredatastudio/blob/master/src/sql/workbench/contrib/notebook/browser/cellViews/code.component.ts#L213